service/src/services/domain/builders/collaboration-callout-published/collaboration.callout.published.notification.builder.ts

```diff
@@ -10,58 +10,58 @@ import { UserPreferenceType } from '@alkemio/client-lib';
 import { User } from '@core/models';
 import { ALKEMIO_URL_GENERATOR } from '@common/enums';
 import { EmailTemplate } from '@common/enums/email.template';
-import { CalloutPublishedEventPayload } from '@alkemio/notifications-lib';
-import { CalloutPublishedEmailPayload } from '@common/email-template-payload';
+import { CollaborationCalloutPublishedEventPayload } from '@alkemio/notifications-lib';
+import { CollaborationCalloutPublishedEmailPayload } from '@common/email-template-payload';
 import { NotificationEventType } from '@alkemio/notifications-lib';
 
 @Injectable()
-export class CalloutPublishedNotificationBuilder
+export class CollaborationCalloutPublishedNotificationBuilder
   implements INotificationBuilder
 {
   constructor(
     private readonly notificationBuilder: NotificationBuilder<
-      CalloutPublishedEventPayload,
-      CalloutPublishedEmailPayload
+      CollaborationCalloutPublishedEventPayload,
+      CollaborationCalloutPublishedEmailPayload
     >,
     @Inject(ALKEMIO_URL_GENERATOR)
     private readonly alkemioUrlGenerator: AlkemioUrlGenerator
   ) {}
   build(
-    payload: CalloutPublishedEventPayload
+    payload: CollaborationCalloutPublishedEventPayload
   ): Promise<NotificationTemplateType[]> {
     const roleConfig: RoleConfig[] = [
       {
         role: 'user',
         preferenceType: UserPreferenceType.NotificationCalloutPublished,
-        emailTemplate: EmailTemplate.CALLOUT_PUBLISHED_MEMBER,
+        emailTemplate: EmailTemplate.COLLABORATION_CALLOUT_PUBLISHED_MEMBER,
       },
     ];
 
     const templateVariables = {
       entityID:
```

Comment:
It is worth using `journeyID` instead of `entityID` everywhere, if we are committed to switching to that terminology.